src/datadog-metrics.ts

```diff
@@ -26,22 +27,28 @@ export function activate(): void {
                 } while (m)
                 STATSD_PATTERN.lastIndex = 0 // reset
             }
-            const decorationType = sourcegraph.app.createDecorationType()
-            setTimeout(() => editor.setDecorations(decorationType, decorations), 200)
+
+            if (sourcegraph.app.activeWindowChanges) {
```

Comment:
This block looks fine, but it should not be contained in the body `decorateEditors()`.  `decorateEditors()` is also probably useless now. You probably want something like this:

```typescript
function decorateEditor(editor: sourcegraph.CodeEditor): void {
    // decoration logic goes here
}

export function activate(context: sourcegraph.extension.Context): void {
    // declare `activeEditor` above
    context.subscriptions.add(
        activeEditor.subscribe(decorateEditor)
    )
}
```